scripts/ci/dev-sso/keycloak.yaml

```diff
@@ -0,0 +1,125 @@
+apiVersion: keycloak.org/v1alpha1
+kind: Keycloak
+metadata:
+  name: sandbox-dev
+  namespace: ${DEV_SSO_NS}
+  labels:
+    sso-toolchain: sandbox-dev
```

Comment:
ideally, we could use `kubesaw` instead of `sandbox`

Reply:
yeah good point! Renamed all the resources in https://github.com/codeready-toolchain/toolchain-cicd/pull/113/commits/f78c9aab726c06c7448fc3d061c85afe2433af78